src/components/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { name } = props;
+
+  return (
+    <button type="button" className="calc-button">
+      { name }
+    </button>
+  );
+};
+
+Button.defaultProps = {
```

Comment:
* [ ] You were not asked to set a default button name per the requirement. All that was needed was prop validation like you have on line 18.